roles/ansible-test/tasks/main.yaml

```diff
@@ -23,14 +23,22 @@
 - name: Set the targets
   set_fact:
     _integration_targets: "{{ ansible_test_integration_targets }}"
+    _test_cmd: "{{ ansible_test_test_command }}"
+
+- name: Set the values for unit tests
+  set_fact:
+    _integration_targets: "tests/unit/modules/network/ovs/{{ _integration_targets }}"
+    _test_cmd: "units"
+    ansible_test_options: ""
+  when: ansible_test_test_command == 'network-unit'
 
 - name: Split targets
   import_tasks: split_targets.yaml
   when: ansible_test_split_in > 1
 
-- name: Run the integration test suite
+- name: Run the test suite
   args:
     chdir: "{{ _test_location }}"
     executable: /bin/bash
   environment: "{{ ansible_test_environment | default({}) }}"
-  shell: "source {{ ansible_test_venv_path }}/bin/activate; {{ ansible_test_executable }} {{ ansible_test_test_command }} {{ ansible_test_options }} --python {{ ansible_test_python }} -vvvv {{ _integration_targets }}"
+  shell: "source {{ ansible_test_venv_path }}/bin/activate; {{ ansible_test_executable }} {{ _test_cmd }} {{ ansible_test_options }} --python {{ ansible_test_python }} -vvvv {{ _integration_targets }}"
```

Comment:
I don't think we need to create _test_cmd here, we can use the value of ansible_test_test_command